app/components/ui/files/publish-modal/component.js

```diff
@@ -219,7 +224,6 @@ export default Ember.Component.extend({
     },
 
     openPackage: function() {
-            
         var win = window.open(this.get('packageUrl'), '_blank');
```

Comment:
Try substituting all your `var` declarations for the correct/updated `let` or `const` on each `.js` file you touch.